src/popup/components/tool.jsx

```diff
@@ -1,18 +1,24 @@
 import PropTypes from 'prop-types';
-import React, { useContext } from 'react';
+import React from 'react';
 import { Link } from 'react-router-dom';
+import browser from 'webextension-polyfill';
 
-import EnvContext from '../env-context';
 import ErrorShape from '../utils/error-shape';
 import TicketShape from '../utils/ticket-shape';
 import Content from './content';
 import Navbar from './navbar';
 import NoTickets from './no-tickets';
 import TicketControls from './ticket-controls';
 
-function Tool({ tickets, errors }) {
-  const { close, openopts } = useContext(EnvContext);
+function close() {
+  window.close();
+}
 
+function openopts() {
```

Comment:
Did you on purpose decide against `openOpts`? Fine, it just took me a few seconds to parse it for the first time 😹

Reply:
I only moved it without (re-)considering the name. Good catch, I'll improve the naming. 💚